asreview/webapp/src/StyledComponents/StyledAlert.js

```diff
@@ -7,7 +7,7 @@ export function ExplorationModeRecordAlert(props) {
       severity="info"
       sx={{ borderBottomRightRadius: 0, borderBottomLeftRadius: 0 }}
     >
-      Labeled as relevant in an earlier study
+      Labeled as relevant in the dataset
```

Comment:
```suggestion
      Labeled as RELEVANT in the dataset
```

What about this? Then makes it easier to distinguish from "Labeled as IRRELEVANT in the dataset".

Reply:
I prefer your initial idea. See screenshots.